src/__tests__/site-apps.ts

```diff
@@ -73,42 +79,6 @@ describe('SiteApps', () => {
     })
 
     describe('constructor', () => {
-        it('sets enabled to true when opt_in_site_apps is true and advanced_disable_decide is false', () => {
-            posthog.config = {
-                ...defaultConfig,
-                opt_in_site_apps: true,
-                advanced_disable_decide: false,
-            } as PostHogConfig
-
-            siteAppsInstance = new SiteApps(posthog)
-
-            expect(siteAppsInstance.enabled).toBe(true)
-        })
```

Comment:
Does `advanced_disable_decide` still do anything? I see that it's used in the code below, so probably worth testing for it somehow.

Reply:
> Does advanced_disable_decide still do anything?

Yes! Added an additional test.